src/lib/ursadb.py

```diff
@@ -43,16 +42,17 @@ def make_socket(self, recv_timeout: int = 2000) -> zmq.Context:
     def query(
         self,
         query: str,
-        taint: Optional[str] = None,
+        taints: List[str] = None,
         dataset: Optional[str] = None,
     ) -> Json:
         socket = self.make_socket(recv_timeout=-1)
 
         start = time.clock()
         command = "select "
-        if taint:
-            taint = taint.replace('"', '\\"')
-            command += f'with taints ["{taint}"] '
+        if taints:
+            taints_str = ", ".join(taints)
+            taints_whole_str = '["' + taints_str + '"]'
+            command += f"with taints {taints_whole_str} "
```

Comment:
this will only work for a single taint, and cause a syntax error when there is more than 1 tag.

Reply:
`["asd", "qwe", "zxc"] ->  ["asd, qwe, zxc"]`